include/ofi_signal.h

```diff
@@ -155,10 +156,11 @@ static inline void fd_signal_reset(struct fd_signal *signal)
 			} else {
 				ofi_atomic_set32(&signal->state, OFI_SIGNAL_SET);
 
-				/* Avoid spinning forever in this highly
-				 * unlikely code path.
-				 */
-				break;
+				if (++retries > 256)
+				{
+					/* Avoid spinning forever. */
+					break;
```

Comment:
reset retries to 0 before the break?

Reply:
There is no code after the loop that the break will terminate, so I'm not sure I understand what benefit setting retries to 0 has. I feel like maybe I'm not seeing something that ought to be obvious?

Reply:
never mind. I was thinking about static variable across multiple calls, which is not the case here,